gymfc.ini

```diff
@@ -3,7 +3,7 @@
 # In most cases this should not be modified
 [DEFAULT]
 
-SetupFile = /usr/share/gazebo-9/setup.sh
+SetupFile = /usr/local/share/gazebo/setup.sh
```

Comment:
I'm not sure the best way to handle this, my Gazebo-10 install placed this in `/home/wil/local/share/gazebo-10/setup.sh`. We've been running into issues changing between machines that have different file paths (that affect the setup file but also where the model.sdf is) and are thinking to provide an option to pass these in to the gymfc constructor as a work around.

Reply:
If you prefer I can patch the file with sed during the docker build like I did with the model.sdf above. 
The ideal way to deal with this is to either have reliable development environment scripts that create the same environment on every machine, or preferred, conduct development and training inside a Docker image that serves as infrastructure as code and runs on a lot of OSs.

Reply:
We're in the process of creating a similar Singularity script to create the environment but this (and most likely for Docker) are most useful during "production" when you need to run a bunch of training scripts automatically. So I think in the end what we should have is:
1) A bash script to install the environment natively on ubuntu (kinda like how Gazebo supplied one)
2) The docker script
3) Singularity script

We'll by default support Gazebo 10 and Dart 6.7.0 for which these scripts will support. We'll bump up these versions periodically after they've been tested. 

Can you drop down the Gazebo version, 11 is still under development. As suggested by the Gazebo install, we'll install to /home/$USER/local, and in the gymfc.ini file well have this set to this file path (assuming the code well interpret the environment variable).